playbooks/provisioning/openstack/README.md

```diff
@@ -584,12 +584,38 @@ This playbook runs against all cluster nodes. In order to help prevent slow conn
 problems, the task is retried 10 times in case of initial failure.
 Note that in order for this example to work in your deployment, your servers must use the RHEL image.
 
+#### Adding extra CAs to the trust chain
+
+```
+---
```

Comment:
Let's please do not repeat code snippets in the docs. Just refer the files included in the repo tree instead.